clusterloader2/pkg/measurement/common/simple/api_responsiveness.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package common
+package simple
```

Comment:
This isn't simple - this is regular SLO measurement.
It shouldn't be part of that bundle.

Reply:
I've assumed that there are two types of metrics:
- simple - which is a standalone metric.
- bundle - aggregation of other metrics.

Even though the `ApiResponsiveness` is not a part of a bundle, IMHO it should be classified as simple.

Reply:
it should be classified as slos (or something like that) together with pod startup time

Reply:
`ApiResponsiveness` and `PodStartupLatency` moved to slos subdirectory.